marketplace/deployer_util/config_helper.py

```diff
@@ -441,7 +694,26 @@ def base64_encoded(self):
 
 
 class SchemaXReportingSecret:
-  """Wrapper class providing convenient access to REPORTING_SECRET property."""
+  """Accesses REPORTING_SECRET property."""
 
   def __init__(self, dictionary):
     pass
+
+
+def _get_or_invalid(dictionary, key, error_msg):
```

Comment:
nit: when I saw the invocations of this function I could not infer what it was doing... can we rename to something more descriptive? like: _get_or_throw[_error]

Reply:
the original name was `_get_or_raise_invalid_schema`, but it was too long and I reduced it to this. I changed this to `_must_get`.